MHVLib/HVMobile/HealthVaultResponse.m

```diff
@@ -18,18 +18,12 @@
 
```

Comment:
I recommend the same cleanup in this class to... Move { to the next line in all cases. Also, it should be safe to remove the iVar definitions in the header file, the private interface definition, and the synthesize statements.